contracts/epochs/FlowEpoch.cdc

```diff
@@ -385,16 +397,17 @@ pub contract FlowEpoch {
                     clusterQCs: clusterQCs,
                     dkgKeys: dkgPubKeys)
 
-            FlowEpoch.epochMetadata[FlowEpoch.currentEpochCounter] = firstEpochMetadata
+            FlowEpoch.epochMetadata[FlowEpoch.currentEpochCounter] = newEpochMetadata
         }
     }
 
     /// Pays rewards to staked nodes and delegators,
     /// Calculates a new token payout for the next epoch
     /// and sets the new payout
-    access(account) fun payAndSetRewards(_ newPayout: UFix64?) {
+    access(account) fun calculateAndSetRewards(_ newPayout: UFix64?) {
```

Comment:
Does this function need to be called manually each epoch? 

From the PR description it sounded to me like we wanted to do this automatically when ending the epoch, but I don't see where that happens.

> Call the calculate rewards function when ending an epoch and store the list

Reply:
I forgot to include the automatic call to it in the heartbeat method, but I've included it in my latest changes